### ISSUE TITLE
Add the expected Hashrateinfo  to the Frequency Tab on Settings

### ISSUE THREAD

**Issue by Bitcoinite** [bitaxeorg/ESP-Miner#444]
Since we know the expected Hashrate for each Frequency (it is even shown on the Dashboard), we should also be able to add this info to Frequency dropdown on the Settings page. This way, users could see what the expected Hashrate for each Frequency setting is, without having to test every setting manually.

<img width="1084" alt="Screenshot 2024-11-02 at 08 36 29" src="https://github.com/user-attachments/assets/d181ca91-2214-43d8-8209-21ffd3f99a13">

**Comment by skot** [comment]
I think technically we could do this. I do worry that it will cause new users to attempt overclocking without understanding the risks (prompting more "my bitaxe is broken" support requests)

**Comment by WantClue** [comment]
> I think technically we could do this. I do worry that it will cause new users to attempt overclocking without understanding the risks (prompting more "my bitaxe is broken" support requests)

I agree on that, technically this might be a nice thing, but increasing the interest in overclocking is probably not what we want to achieve.
